uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/ProblemsViewTest.java

```diff
@@ -65,7 +66,7 @@
 	public void testProblemsOnGlobalElements() {
 		ProjectFactory.newProject(PROJECT_NAME).deploymentType(STANDALONE).runtimeType(KARAF).template(EMPTY_SPRING)
 				.create();
-		new CamelProject(PROJECT_NAME).openCamelContext(CAMEL_CONTEXT_XML);
```

Comment:
There is no need to open the Camel Context File ... It is opened automatically after project creation process is done